modules/conversantAnalyticsAdapter.js

```diff
@@ -3,16 +3,20 @@ import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
 import CONSTANTS from '../src/constants.json';
 import {getGlobal} from '../src/prebidGlobal.js';
 import adapterManager from '../src/adapterManager.js';
-import {logInfo, logError, logMessage, deepAccess, isInteger} from '../src/utils.js';
+import {logInfo, logWarn, logError, logMessage, deepAccess, isInteger} from '../src/utils.js';
+import {getRefererInfo} from '../src/refererDetection.js';
 
 const {
-  EVENTS: { AUCTION_END, AD_RENDER_FAILED, BID_TIMEOUT, BID_WON }
+  EVENTS: { AUCTION_END, AD_RENDER_FAILED, BID_TIMEOUT, BID_WON, BIDDER_ERROR }
 } = CONSTANTS;
+// STALE_RENDER, TCF2_ENFORCEMENT would need to add extra calls for these as they likely occur after AUCTION_END?
```

Comment:
Yes, STALE_RENDER would occur after AUCTION_END.   There could be several STALE_RENDER after an auction has ended.